packages/terra-date-picker/src/react-datepicker/date_utils.js

```diff
@@ -67,7 +67,13 @@ export function cloneDate (date) {
 
 export function parseDate (value, { dateFormat, locale }) {
   const m = moment(value, dateFormat, locale || moment.locale(), true)
-  return m.isValid() ? m : null
```

Comment:
Not suggesting it now, but should we someday merge date_utils.js and DateUtil.js?

Reply:
I agree. Some consumers are using this directly so we should prepend the new file name with an underscore `_` to denote that it's private and not intended for reusable.

Reply:
based on previous discussions, i believe we decided to use a `private` directory to denote files that are private. we've followed through in application base https://github.com/cerner/terra-application/tree/master/src/application-base/private